var/spack/repos/builtin/packages/py-salib/package.py

```diff
@@ -14,6 +14,7 @@ class PySalib(PythonPackage):
 
     maintainers = ['schmitts']
 
+    version('1.4.4', sha256='50a6459088700f55261a683752818530d14ede30cece2c324ac94d4b9e288b6d')
     version('1.4.0.1', sha256='dbf6e865af9f3be82a79cf64889ed66d6d3b6803f0c22a242a112876789d49e7')
 
     depends_on('py-setuptools@38.3:', type=('build', 'run'))
```

Comment:
Can you update the dependencies? I see a new build dep on setuptools-scm and build/run deps on importlib-metadata and wheel (wheel should not be in install_requires since it isn't required for build or run...)

Reply:
Will do.

Reply:
The wheel dependency was unnecessary added on my request but it turned out it only helped for an outdated pip version.

Reply:
Doesn't setuptools automatically download and install any packages in `install_requires` that aren't already installed (including wheel)?

Reply:
I'll check.

Reply:
No reply from the devs (https://github.com/SALib/SALib/pull/440#issuecomment-899581387). I've updated my Spack PR to include wheel for the time being.